dglc/dglc_datamode_noevolve_mod.F90

```diff
@@ -638,13 +638,8 @@ subroutine dglc_datamode_noevolve_restart_read(model_meshes, restfilem, &
        call ESMF_VMGetCurrent(vm, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
        if (my_task == main_task) then
-          write(logunit,'(a)') trim(subname)//' restart filename from rpointer'
-          inquire(file=trim(rpfile)//trim(inst_suffix), exist=exists)
-          if (.not.exists) then
-             write(logunit, '(a)') trim(subname)//' ERROR: rpointer file does not exist'
-             call shr_sys_abort(trim(subname)//' ERROR: rpointer file missing')
-          endif
-          open(newunit=nu, file=trim(rpfile)//trim(inst_suffix), form='formatted')
+          write(logunit,'(a)') trim(subname)//' restart filename from rpointer '//trim(rpfile)
```

Comment:
Don't you have to still inquire if this file exists? Why did you remove this?

Reply:
No - this is now done in get_shr_rpointer_name.
https://github.com/ESCOMP/CESM_share/pull/53/files#diff-812ce9a5da247fa8b7adf0aa10cbe625ce01376a61c6e8af00a92d9cf6adb914R879